src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -820,7 +846,7 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
     auto run_passes = [&](const CNNNetwork& network, bool runBeforeCopy, bool lowPrecision) {
         auto passes = make_shared<PassManager>(PassManagerSettings{runBeforeCopy, lowPrecision}, network);
         passes->registerPass<RemoveConstPass>();
-        if (!isNgraphPassesUsed) {
+        if (!is_ngraph_passes_used) {
```

Comment:
Maybe it would be good to move this lambda to separate file or method the same as TransformModel was introduced?
It would increase readability of LoadNetwork().

Reply:
Moved to separate function and separate file